js/app.js

```diff
@@ -70,12 +63,122 @@ function getDate() {
 }
 
 function Controls() {
-    var LocationsDropdown = {
+    let showModal = false;
+    let canteens = [];
+
+    function openStreetMap() {
+        const greenIcon = new L.Icon({
+            iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-green.png',
+            shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
+            iconSize: [25, 41],
+            iconAnchor: [12, 41],
+            popupAnchor: [1, -34],
+            shadowSize: [41, 41]
+        });
+
+        return {
+            oncreate: function (vnode) {
+                const map = L.map(vnode.dom)
+                    .setView([48.15, 11.55], 10); // coordinates for munich
+                L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+                    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
```

Comment:
We have to credit OpenStreetMap to use it
> [...] You are free to copy, distribute, transmit and adapt our data, as long as you credit OpenStreetMap and its contributors. [...]

https://www.openstreetmap.org/copyright

Reply:
I used the example code from leaflet, which includes the Copyright mention here in line 84, don't you think, this is sufficient?
It looks like this on the webpage (bottom right corner):
<img width="636" alt="Bildschirmfoto 2021-11-20 um 17 36 41" src="https://user-images.githubusercontent.com/1690395/142734062-6b7f463b-4346-4817-8cc7-f57429484038.png">

Reply:
Aha! Overlooked that. that's fine 👍.